cmd/seeder/v2/2_users.go

```diff
@@ -14,11 +15,10 @@ import (
 )
 
 func main() {
-	// err := config.LoadEnv()
-	// if err != nil {
-	// 	log.Println(err)
-
-	// }
+	err := config.LoadEnv()
+	if err != nil {
+		log.Println(err)
```

Comment:
Maybe we should return and let user know with an error if the env variables are not loaded correctly since next we are trying to connect with the DB using the env variables.

Reply:
Yeah, here it should `Fatal` out, as this is `main` func and without `db` we wouldn't be able to fetch the `users` anyway